_rules/auto-play-audio-has-control-mechanism-4c31df.md

```diff
@@ -0,0 +1,226 @@
+
+## Expectation 1
+
+For each test target a [mechanism](https://www.w3.org/TR/WCAG21/#dfn-mechanism) is provided to pause or stop the audio, or turn the audio volume off independently from the overall system volume control.
+
+## Expectation 2
+
+The [mechanism](https://www.w3.org/TR/WCAG21/#dfn-mechanism) to pause or stop or turn the audio volume off is [visible](#visible), has an [accessible name](#accessible-name), and is [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+## Assumptions
+
+*There are currently no assumptions*
+
+## Accessibility Support
+
+The native `<video>` and `<audio>` controls in several browser and assistive technology combinations are not keyboard accessible and the `<video>` or `<audio>` element itself may not be anounced. Authors are recomended to use custom controls for keyboard navigation and cross browser accessibility support in general.
+
+## Background
+
+- [Understanding Success Criterion 1.4.2: Audio Control](https://www.w3.org/WAI/WCAG21/Understanding/audio-control.html)
+- [Failure of Success Criterion 1.4.2 for absence of a way to pause or stop an HTML5 media element that autoplays](https://www.w3.org/WAI/WCAG21/Techniques/failures/F93)
+- [G170: Providing a control near the beginning of the Web page that turns off sounds that play automatically](https://www.w3.org/WAI/WCAG21/Techniques/general/G170)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+The `<audio>` element has a [mechanism](https://www.w3.org/TR/WCAG21/#dfn-mechanism) to pause or stop or turn the audio volume off.
+
+``` html
+  <audio src="../test-assets/moon-audio/moon-speech.mp3" autoplay="true" controls></audio>
+```
+
+#### Passed Example 2
+
+The `<video>` element has a [mechanism](https://www.w3.org/TR/WCAG21/#dfn-mechanism) to pause or stop or turn the audio volume off.
+
+``` html
+ <video autoplay="true" controls>
+  <source src="../test-assets/rabbit-video/video.mp4" type="video/mp4" />
+  <source src="../test-assets/rabbit-video/video.webm" type="video/webm" />
+</video>
+```
+
+#### Passed Example 3
+
+The `<video>` element has a custom [mechanism](https://www.w3.org/TR/WCAG21/#dfn-mechanism) to pause or stop or turn the audio volume off.
+
+``` html
+<head>
+<style>
+button {color: #000;}
+button:hover {cursor: pointer;	cursor: pointer; background-color: grey;  color: white;}
+</style>
+</head>
+<body>
+	<div id="video-container">
+		<!-- Video -->
+		<video id="video" autoplay="true">
+		 <source src="https://act-rules.github.io/test-assets/rabbit-video/video.mp4" type="video/mp4">
+	   	 <source src="https://act-rules.github.io/test-assets/rabbit-video/video.webm" type="video/webm" />
+		</video>
+		<!-- Video Controls -->
+		<div id="video-controls">
+			<button type="button" id="play-pause" class="play">Play</button>
+			<button type="button" id="mute">Mute</button>
```

Comment:
This needs to have a script, no?

Reply:
added `<script src="no-autoplay.js"></script>` reference for https://github.com/act-rules/act-rules.github.io/blob/a663d2df70c3ac2d71d47072de9f8d8a05bdbe61/assets/js/no-autoplay.js